packages/cli/src/index.ts

```diff
@@ -19,8 +19,8 @@ import pkg from '../package.json';
 
 const cli = cac(pkg.name.split('/')[1]);
 
-cli.command('login <instanceUrl>', 'log in to a Frontify instance')
-    .option('-i, --instance <instanceUrl>', '[string] url of the Frontify instance')
+cli.command('login [instanceUrl]', 'log in to a Frontify instance')
+    .option('-i, --instance [instanceUrl]', '[string] url of the Frontify instance')
```

Comment:
I think in this case `<instanceUrl>` is needed, because if we pass the `--instance` option, we need to pass the url as well.